build.rs

```diff
@@ -16,12 +16,47 @@ fn rbconfig(key: &str) -> Vec<u8> {
     config.stdout
 }
 
+fn use_static() {
+    let ruby_libs = rbconfig("LIBS");
+    let libs = String::from_utf8_lossy(&ruby_libs);
+
+    // Ruby gives back the libs in the form: `-lpthread -lgmp`
+    // Cargo wants them as: `-l pthread -l gmp`
```

Comment:
For reference, here is the error I get when attempting to build without spacing out the -l arg:

```
error: Only `-l` and `-L` flags are allowed in build script of `ruby-sys v0.2.20 (file:///home/ianks/Documents/FOSS/ruby-sys)`: `-lpthread -ljemalloc -lgmp -ldl -lcrypt -lm`
```